packages/python/fix_imports.py

```diff
@@ -11,25 +11,22 @@
 else:
     from typing_extensions import TypedDict
 
-import click
-
+import argparse
 
 class ProtobufFilePathInfo(TypedDict):
     dir: Path
     path: Path
     rel_path: Path
 
-
-@click.command()
-@click.option("--dry", is_flag=True, show_default=True, default=False,
-              help="Do not write out the changes to the files.")
-@click.argument("root_dir", type=click.Path(exists=True))
 def fix_protobuf_imports(root_dir, dry):
     """
       A script to fix relative imports (from and to nested sub-directories) within compiled `*_pb2*.py` Protobuf files.
     """
 
     root_dir = Path(root_dir)
+    if not root_dir.is_dir():
+        print(f"Error: The specified root directory '{root_dir}' does not exist.")
```

Comment:
It is better to raise a suitable exception instead of printing. Try something like:
````
raise argparse.ArgumentError("rout_dir", "<error message>")
```